style/style.css

```diff
@@ -383,7 +383,7 @@ body > footer {
 		top: 0;
 		bottom: 0;
 		z-index: -1;
-		width: 50vw;
+		width: 49.6vw;
```

Comment:
I would think so but here's what I see.
Total window size - 1366px
Each `:after` element taking 683px themselves. 
We still have the border left out.

However what's strange is that even after removing the margin and border on parent element, it still occupies some space beyond the window width! 

![image](https://cloud.githubusercontent.com/assets/313363/11790191/70e49dd4-a268-11e5-9b06-634a1124c4d6.png)